assets/parquet/main.py

```
@@ -0,0 +1,154 @@
+        """
+        res = self.hive_execute(f"SELECT * from {PARQUET_TABLE_NAME} WHERE {query.strip()[1:-1]}").strip()
+        if not res:
+            return 0
+        return res.count('\n') + 1
+
+    def clear_cache(self):
+        """
+        Clears the cache within the docker container for cold run
+        """
+        self.docker_execute("sync")
+        self.docker_execute("echo 1 >/proc/sys/vm/drop_caches", check=False, shell=True)
```

Ruff (0.12.2)

Function call with `shell=True` parameter identified, security issue

(S604)